src/main/java/ee/tuleva/onboarding/payment/recurring/RecurringPaymentService.java

```diff
@@ -65,6 +65,8 @@ public PaymentLink getPaymentLink(PaymentData paymentData, Person person) {
               + "&MakseSagedus=3" // Monthly
               + "&MakseEsimene="
               + tenthDayOfMonth(LocalDate.now(clock));
+          case TULUNDUSUHISTU -> throw new IllegalArgumentException(
+              "Can not make a recurring payment to a given bank!");
```

Comment:
"Recurring payments to the specified bank are not supported"

Reply:
Thanks